File: app/src/main/java/apps/jizzu/simpletodo/utils/MyApplication.java
package apps.jizzu.simpletodo.utils;

import android.app.Application;

/**
 * Class that helps to get the current status of MainActivity.
 */
public class MyApplication extends Application {

    private static boolean mActivityVisible;

    public static boolean isActivityVisible() {
        return mActivityVisible;
    }

    public static void activityResumed() {
        mActivityVisible = true;
    }

    public static void activityPaused() {
        mActivityVisible = false;
    }
}

File: app/src/main/java/apps/jizzu/simpletodo/utils/MyApplication.kt
package apps.jizzu.simpletodo.utils

import android.app.Application

/**
 * Class that helps to get the current status of MainActivity.
 */
class MyApplication : Application() {
    companion object {

        var isActivityVisible: Boolean = false
            private set

        fun activityResumed() {
            isActivityVisible = true
        }

        fun activityPaused() {
            isActivityVisible = false
        }
    }
}
